### DISCUSSION TITLE
DOSXYZnrc Gui Job failed

### DISCUSSION THREAD

**Discussion by darkblue3699** [nrc-cnrc/EGSnrc#847] [Questions] [answered]
Hello.
I have a problem running  DOSXYZnrc GUI. After loading the  16MVp_h20_phantom_beamsource_example and compiling sucessfully, but  I failed to excute the running process.  

Thanks for help.

![image](https://user-images.githubusercontent.com/42909701/153328858-dad6c5db-f42e-4b18-8ef7-58ae096bfeb0.png)

![image](https://user-images.githubusercontent.com/42909701/153328967-78271017-6b5a-4fc0-a7e3-c443fcf3dacf.png)

**Comment by rtownson** [comment, marked as answer]
Unfortunately the GUI doesn't always give all the output. If you run on the command line, you will see more text that describes the problem.

This example requires an additional step to be functional, because it uses `BEAM_EX16MVp` as a source of particles. You need to compile that accelerator as a shared library. To do this, go into `egs_home/BEAM_EX16MVp` on the command line and type `make library` (or `mingw32-make` on windows). Then the example should run.

**Reply by darkblue3699** [reply]
Thanks for your help. Fortunately, I ran the job on the commnd line sucessfully.
